src/main/resources/assets/admin/common/styles/api/input-common.less

```diff
@@ -183,6 +183,12 @@
         }
       }
 
+      &.max-length-limited {
+        + .help-text {
+          padding-right: 175px;
+        }
+      }
+
       &.max-length-limited.show-counter {
         .separator {
           display: inline;
```

Comment:
As you can see, there's already selector for `&.max-length-limited` below, so you need to group the two:

```CSS
&.max-length-limited {
  + .help-text {
    padding-right: 175px;
  }

  &.show-counter {
    .separator {
       display: inline;
    }
  }
}
```